Tests/L0/SonarQubePostTest/PRCA/InvokeTests.ps1

```diff
@@ -0,0 +1,79 @@
+[CmdletBinding()]
+param()
+
+
+$source = @"
+
+using System.Collections;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using System.Threading;
+using System.Linq;
+using System;
+
+namespace Test
+{
+    // Used to test the reflection invokation helper
```

Comment:
Missing coverage of the cases mentioned above (mostly around polymorphism with/without overloads).